### ISSUE TITLE
Crash in ES build

### ISSUE THREAD

**Issue by lig** [ant-design/ant-design#7123]
### Version
2.12.4

### Environment
Fedora 26, node v8.1.3, npm 5.3.0

### Reproduction link
[https://github.com/ant-design/ant-design](https://github.com/ant-design/ant-design)

### Steps to reproduce
Build es6 project using configuration from https://github.com/facebookincubator/create-react-app

### What is expected?
Clean build

### What is actually happening?
```
./~/antd/es/select/index.js
20:42-59 "export 'SelectPropTypes' (imported as 'RcSelectPropTypes') was not found in 'rc-select'
```

**Comment by afc163** [comment]
Fixed in rc-select@6.8.9, please try to reinstall `node_modules`.

**Comment by lig** [comment]
@afc163 I actually did it several times to be sure that it reproduces before you've commented. Now it works. Thanks!
